CHANGELOG.md

```diff
@@ -3274,7 +3274,7 @@ Other:
 
 
 ## 1.1.2
-> Released on 2021/04/09
+> Released on 2021/04/13

```

Comment:
@lukidzi are the dates right?

Reply:
Yep it's due to: https://github.com/kumahq/ci-tools/pull/56